poc.md

```diff
@@ -0,0 +1,155 @@
+# Proof of Concept for Layer 2 Networking in Orchestrator
+
+This file contains notes and code for proving the networking design as defined in the [EMT-S Scale Provisioning design
+proposal](https://github.com/open-edge-platform/edge-manageability-framework/blob/4d801d24d402871ee664fbedcd3f964d43a784b2/design-proposals/emts-scale-provisioning.md?plain).
+
+## Deployment
+
+### Prerequisites
+
+1. On-premise Orchestratror fully deployed on the system
+   1. Edge Network is configured with a bridge interface `virbr1`
+1. `kubectl` is configured with root permissions to manage the Kubernetes cluster
+1. `nmap` is installed on the system to verify network connectivity
+   1. On Ubuntu, install with `sudo apt install nmap`
+
+### Steps
+
+1. Create a pod the deploys a DHCP server with host networking
+
+```shell
+cat <<EOF | kubectl apply -f -
+apiVersion: v1
+kind: Pod
+metadata:
+  name: dhcp-server
+  namespace: default
+spec:
+  hostNetwork: true  # Use the host's network namespace
+  containers:
+  - name: dhcp-server
```

Comment:
This is in line with my PR: https://github.com/open-edge-platform/infra-charts/pull/53/files